examples/example_wallet_electrum/src/main.rs

```diff
@@ -92,5 +97,29 @@ fn main() -> Result<(), anyhow::Error> {
     client.transaction_broadcast(&tx)?;
     println!("Tx broadcasted! Txid: {}", tx.compute_txid());
 
+    println!("=== Performing Partial Sync ===");
+    std::io::stdout().flush().expect("must flush");
```

Comment:
Is this strictly needed?

Reply:
I do think so because the progress update may appear before the `performing partial sync` message if it is not flushed (when using `eprintln`). I will update this, so it is consistent.